cmd/showcommit.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/qri-io/ioes"
+	"github.com/qri-io/qri/lib"
+	"github.com/spf13/cobra"
+)
+
+// NewShowCommitCommand creates a new `qri search` command that searches for datasets
+func NewShowCommitCommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
+	o := &ShowCommitOptions{IOStreams: ioStreams}
+	cmd := &cobra.Command{
+		Use:   "showcommit",
+		Short: "Show information about a commit",
+		Long: `
+The command showcommit will show information about a commit, specifically what
+components were added or removed, at a specific version. This is analagous to
+the status command, except only available for dataset versions in history.
+
+  # show information about the head commit
+  qri showcommit me/dataset_name`,
```

Comment:
plz move this into `Example:`, again following guidelines. Examples should be
* Indented with two spaces.
* Example descriptions start with #.
* Lines that should be typed start with $.